client/src/app/modules/components/sections/section-view/section-view.component.ts

```diff
@@ -51,6 +51,15 @@ export class SectionViewComponent implements OnChanges {
             name: 'Actions',
             cellTemplate: 'actionsCell',
             actions: (row: MeetingWithLesson<MeetingType.RECITATION> & { getAccessCode: () => string }) => [
+                {
+                    text: 'Join with Zoom',
+                    if: (row.meetingTime.meetingLink !== null && row.meetingTime.meetingLink !== ''),
```

Comment:
Let's just simplify this to `if: !!row.meetingTime.meetingLink`, we already know it won't be `''` or it wouldn't pass validation and thus be null.